weave/flow/integrations/mistral/mistral.py

```diff
@@ -0,0 +1,109 @@
+import importlib
+
+import typing
+
+import weave
+from weave.trace.op_extensions.accumulator import add_accumulator
+from weave.trace.patcher import SymbolPatcher, MultiPatcher
+
+if typing.TYPE_CHECKING:
+    from mistralai.models.chat_completion import (
+        ChatCompletionStreamResponse,
+        ChatCompletionResponse,
+    )
+
+
+def mistral_accumulator(
```

Comment:
This function is manual now, but it might be able to share a lot of logic with OpenAI if they use similar delta patterns